EIPS/eip-3670.md

```diff
@@ -65,45 +63,48 @@ This change poses no risk to backwards compatibility, as it is introduced at the
 
 ### Contract creation
 
-Each case should be tested for creation transaction, `CREATE` and `CREATE2`.
+Each case should be tested by submitting an EOF container to EOF contract creation (as specced out in a separate EIP).
 
-- Invalid initcode
-- Valid initcode returning invalid code
-- Valid initcode returning valid code
+- EOF container with an invalid code at zeroth code section
+- EOF container with an invalid code at code sections placed in various positions
```

Comment:
I don't understand this.

Reply:
Invalid code can be found in code section 0 or some other, test cases should cover that. It is phrased clunkily, I'll rework this. How about

```
- EOF container with invalid code in one of its code sections (test various code section indices)
```
?

Reply:
I'm a terrible wordsmith here, how about fa60e768 ?